sdk/keyvault/Azure.Security.KeyVault.Administration/src/KeyVaultAccessControlClientOptions.cs

```diff
@@ -14,7 +14,7 @@ public class KeyVaultAccessControlClientOptions : ClientOptions
         /// <summary>
         /// The latest service version supported by this client library.
         /// For more information, see
-        /// <see href="https://docs.microsoft.com/en-us/rest/api/keyvault/key-vault-versions"/>.
+        /// <see href="https://docs.microsoft.com/rest/api/keyvault/key-vault-versions">Key Vault versions</see>.
```

Comment:
wasn't sure if I should just remove this or if there's a better replacement. There's no such article.

Reply:
Seems it used to exist, and is still referenced in the REST API docs:

![image](https://user-images.githubusercontent.com/1532486/85094201-0a5eec00-b1a3-11ea-9931-3e7c76979113.png)

I'll contact the service PM about this, but probably leave it for now. I'll ask they keep the existing URL. In the worst case, at least it goes to the KV home page.